crates/sui-core/src/checkpoints/checkpoint_executor/metrics.rs

```diff
@@ -25,6 +26,9 @@ pub struct CheckpointExecutorMetrics {
     // TODO: delete once users are migrated to non-Mysten histogram.
     pub last_executed_checkpoint_age_ms: MystenHistogram,
     pub checkpoint_executor_validator_path: IntGauge,
+
+    pub stage_wait_duration_ns: IntCounterVec,
```

Comment:
Would it make more sense for these to be histograms?

Reply:
I'm not sure that would useful? We can always add them later